docs/api-overview.md

```diff
@@ -24,9 +24,30 @@
 
 # API documentation
 
-The Eclipse OpenJ9 VM provides supplementary application programming interfaces and extensions, which can be used to improve performance, assist with problem determination, or help monitor and manage the OpenJ9 VM.
+The Eclipse OpenJ9 VM provides supplementary application programming interfaces and extensions, which can be used to improve performance, assist with problem determination, or help monitor and manage the OpenJ9 VM. The documentation also includes links to the API documentation for the Java&trade; SE and JDK reference implementation.
+
+
+## Native data operations
+
+If your Java application manipulates native data, the Data Access Accelerator API package (`com.ibm.dataaccess`) can help improve application performance. Classes are available for manipulating packed decimal data, converting between decimal data types, and marshalling Java binary types to and from byte arrays.
+
+## GPU acceleration
+
+You can improve the performance of your applications by offloading certain processing functions from your processor (CPU) to a graphics processing unit (GPU). If your application contains code that would benefit from parallel processing, you can use the CUDA4J API package (`com.ibm.cuda`) to specify in your code when to offload processing to the GPU. You can also use the GPU API package (`com.ibm.gpu`)  to accelerate certain Java functions, such as `sort` operations.
+
+## Problem determination
+
+The JVM diagnostic utilities API package (`com.ibm.jvm`) provides classes for controlling dump, log, and trace operations.
+
+The Diagnostic Tool Framework for Java (DTFJ) API packages (`com.ibm.dtfj.*`) allow custom applications to be written that can access a wide range of information in a system dump or a Java dump.
+
+## Monitoring and management
+
+The shared classes API package (`com.ibm.oti.shared`) provides a large number of classes for managing permissions, finding and storing classes and byte data, and obtaining statistics about a shared classes cache. Classes are also available to add shared class caching to a custom class loader implementation.

```

Comment:
For "Classes are also available to add shared class caching to a custom class loader implementation", 
probably better to say "Classes are also available to enable class sharing for a custom class loader."

Reply:
Thanks. Updated.